websites/C/CliffsNotes/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Gurrman375D",
+      "id": "446528923535278082"
+    },
+    "url": "www.cliffsnotes.com",
+    "description": {
+      "en": "CliffsNotes study guides are written by real teachers and professors, so no matter what you're studying, CliffsNotes can ease your homework headaches and help you score high on exams."
+    },
+    "service": "CliffsNotes",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/qLjfR9U.png",
+    "thumbnail": "https://i.imgur.com/qLjfR9U.png",
```

Comment:
Please change the thumbnail to a promotional image or a screenshot of the website

Reply:
Don't know how that got messed up